addons/account/static/src/components/actionable_errors/actionable_errors.js

```diff
@@ -0,0 +1,53 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { Component, xml } from "@odoo/owl";
+import { standardFieldProps } from "@web/views/fields/standard_field_props";
+import { useRecordObserver } from "@web/model/relational_model/utils";
+
+export class ActionableErrors extends Component {
+    static props = { ...standardFieldProps };
+    static template = xml`
+    <t t-name="account.ActionableErrors">
+        <t t-if="errorsData">
+            <div class="o_actionable_errors" role="alert">
+                <t t-foreach="errorsData" t-as="error" t-key="error">
```

Comment:
Sorry, the `error` key here is a dictionary key.
If I put `error_index` here, I lose the whole reactivity of it.
Since Josse wants to put it in a compute, we don't need to re-render it whole when it changes.
```js
{
    'partner_missing_address': // <---
         {
             'message': ...,
             ...
         }         
}
```

Reply:
ok, dict keys should be unique